README.md

```diff
@@ -44,6 +44,7 @@ You'll also need to run
 
 ```
 pip install -r requirements-dev.txt
+pre-commit
```

Comment:
Is the placement of this command here intentional? I don't believe the user needs to explicitly run `pre-commit` here to install anything; is it here to have them check that it is working?

If that's the case, I'd just call it out in a separate code block as a sort of FYI.

Reply:
From https://pre-commit.com/ the user does need to run `pre-commit` or `pre-commit install` to initialize it after installing the pip package, yeah.

Reply:
Oh, good to know. Thanks for the clarification!